deployer/src/deployer/search/__init__.py

```diff
@@ -276,7 +276,8 @@ def analyze(
             print()
             for key in keys:
                 print(f"{key:{longest_key + 1}} {token[key]!r}")
-
+    elif not analysis:
+        print("No tokens found!")
```

Comment:
This is quite unimportant but I found it useful when you're CLI testing the analyzer and it yields `tokens == []`.